src/main/resources/credit_cards.avro

```diff
@@ -0,0 +1,152 @@
+                },
+                { "card_id": 10,
+                  "card_number": 4242424242424242, 
+                  "cvv": 190, 
+                  "expiration_date": "09/27"
+                },
+                { "card_id": 11,
+                  "card_number": 4000056655665556, 
+                  "cvv": 729, 
+                  "expiration_date": "09/26"
+                },
+                { "card_id": 12,
+                  "card_number": 5555555555554444, 
+                  "cvv": 723, 
+                  "expiration_date": "08/27"
+                },
+                { "card_id": 13,
+                  "card_number": 2223003122003222, 
+                  "cvv": 723, 
+                  "expiration_date": "08/26"
+                },
+                { "card_id": 14,
+                  "card_number": 5200828282828210, 
+                  "cvv": 723, 
+                  "expiration_date": "08/27"
+                },
+                { "card_id": 15,
+                  "card_number": 5105105105105100, 
+                  "cvv": 723, 
+                  "expiration_date": "07/27"
+                },
+                { "card_id": 16,
+                  "card_number": 6200000000000005, 
+                  "cvv": 723, 
+                  "expiration_date": "06/27"
+                },
+                { "card_id": 17,
+                  "card_number": 3056930009020004, 
+                  "cvv": 723, 
+                  "expiration_date": "05/27"
+                },
+                { "card_id": 18,
+                  "card_number": 4000000760000002, 
+                  "cvv": 723, 
+                  "expiration_date": "04/27"
+                },
+                { "card_id": 19,
+                  "card_number": 4000001240000000, 
+                  "cvv": 723, 
+                  "expiration_date": "03/27"
+                },
+                { "card_id": 20,
+                  "card_number": 4000004840008001, 
+                  "cvv": 723, 
+                  "expiration_date": "02/27"
+                },
+                { "card_id": 21,
+                  "card_number": 4000002500003155, 
+                  "cvv": 723, 
+                  "expiration_date": "02/26"
+                },
+                { "card_id": 22,
+                  "card_number": 4000002760003184, 
+                  "cvv": 723, 
+                  "expiration_date": "02/27"
+                },
+                { "card_id": 23,
+                  "card_number": 4000008260003178, 
+                  "cvv": 723, 
+                  "expiration_date": "01/26"
+                },
+                { "card_id": 24,
+                  "card_number": 40000038000004468,
+                  "cvv": 723,
+                  "expiration_date": "01/27"
+                },
+                { "card_id": 25,
+                  "card_number": 40000535600000118,
+                  "cvv": 723,
+                  "expiration_date": "05/27"
+                }
+            ]
```

Comment:
IIRC the options file feature works exactly the same as the inline options feature, it just allows you to extract the list of options into a separate, reusable location. I think that that should satisfy the constraints you outline here as long as they're already satisfied by the inline options list.

Not a huge deal either way, just thought it might be useful if you want the same list of credit cards to be reusable across different generated data sets.

Reply:
@C0urante I don't think there's a use case that would require that change.  I'm fine with it being embedded as it is now, if you are.

Reply:
Sounds good 👍